### ISSUE TITLE
Sign in with Google temporarily disabled for this app

### ISSUE THREAD

**Issue by ofersabo** [odeke-em/drive#1087]
After installing and launching the gdrive. 
When I try to get the needed authentication I get this error from google servers. 

"Sign in with Google temporarily disabled for this app
This app has not been verified yet by Google in order to use Google Sign In."

**Comment by jchprj** [comment]
I tried [gdrive](https://github.com/gdrive-org/gdrive) and going to try this one but see this same issue, maybe the [workaround](https://github.com/gdrive-org/gdrive/issues/546#issuecomment-626208303) in the gdrive issues could be a workaround for this either.

**Comment by noamTzi** [comment]
Hi did you solve this somehow without the suggested workaround?
I get the same error, and yet I can push files to the google drive but not to pull from there.

**Comment by jchprj** [comment]
No, I havn't tried any more. I may consider rclone in the future as it supports more cloud providers.
